### PR TITLE
Fixed -SessionVariable scoping issue

### PR DESCRIPTION
**Pull request by jangins101**
In the _Invoke-WebRequest_ and _Invoke-RestMethod_ methods, when the _SessionVariable_ parameter is used, it appears that those base methods set the variable based on the parent scope, which would be inside the module method wrappers. Because of this, the variable isn't available to the original calling scope. Added a line that will take the value of the variable and scope it 2 levels up (which seems to be the minimum to get out of the module and into where the calling script/user can access it).

### CODE REVIEW

**Comment by Jaykul** [comment]
I've apparently not been getting notifications properly about comments/issues on my projects ...

Have you tested this fix when calling these cmdlets from script files, or from other modules?  Module scope is weird, not sure if this will work ...

**Comment by elovelan** [comment]
@Jaykul, I've confirmed this works when called from a script. I have a few refactors for readability ready but probably best to merge this to give @jangins101 the credit

TL;DR: `-Scope 2` is necessary because this is called within the `End` script block which's parent scope is the module itself (i.e. `-Scope 1` refers to the module)
